src/pokemon_jump_3.c

```diff
@@ -0,0 +1,167 @@
+#include "global.h"
+#include "link_rfu.h"
+#include "pokemon_jump.h"
+
+struct MonInfoPacket
+{
+    u8 id; // packet id
+    u16 species;
+    u32 personality;
+    u32 otId;
+};
+
+void sub_8149A6C(struct PokemonJump1_MonInfo *arg0)
+{
+    struct MonInfoPacket packet;
+    packet.id = 1,
+    packet.species = arg0->species,
+    packet.otId = arg0->otId,
+    packet.personality = arg0->personality,
+        RfuPrepareSend0x2f00(&packet);
```

Comment:
What's the purpose of the extra indent? 
And I don't see the advantage of using comma here

Reply:
Ugh... When pressing cmd+tab on my mac, sometimes the "cmd" doesn't take before the "tab", and this happens.